src/views/simulation/resultats.vue

```diff
@@ -65,9 +65,9 @@
           Votre simulation n'a pas permis de découvrir de nouveaux droits.
         </h2>
         <p class="aj-results-intro">
-          Si vous êtes dans une situation difficile, d'<router-link to="/sos">
-            autres solutions existent </router-link
-          >.
+          Nous mettons à jour régulièrement le simulateur en ajoutant de
+          nouvelles aides. N'hésitez pas à faire une simulation dans les
+          prochains mois.
         </p>
```

Comment:
On peut peut-être supprimer tout le paragraphe.
Cela étant dit, ta reformulation donne envie de permettre aux personnes de laisser leur email et qu'on réévalue leur droit aux nouvelles aides ajoutées en fur et à mesure.

Reply:
Je pensais le supprimer mais on a plusieurs fois mentionné le fait de donner envie aux personnes de revenir. Ca m'a fait penser à un point où on communique pas forçément sur l'actualité du simulateur
https://trello.com/c/TgrzHGoH/488-ajout-dune-page-dactualit%C3%A9-du-simulateur-pour-tenir-au-courant-des-nouveaut%C3%A9s-et-des-nouvelles-aides-ajout%C3%A9es-dans-le-simulateur
Un point qu'on pourra discuter en point de priorisation

Reply:
En fait il y a de la comm' qui est faite par d'autres personnes (1J1S) sur le simulateur (avec une efficace discutable). Je trouve dommage de ne pas offrir la possibilité aux personnes de ne pas avoir à toute ressaisir.

Reply:
J'ajoute une carte dans à prioriser : https://trello.com/c/aPrDoSo1/501-permettre-la-reprise-de-simulation :)